app/code/Mykhailok/SupportChat/view/frontend/web/js/model/message.js

```diff
@@ -0,0 +1,89 @@
+define([
```

Comment:
@mykhailok-dv 
1) This model represents the whole chat, not just a single message. It should be called `chat.js` or `chatMessages.js`, but not `messages.js`
2) There is no need to have this model - rather save data in the proper format inside the localStorage (customerData).
3) There is no need to have a model for this, because nobody requires it. It is to early to make this extendable, because other scripts can just take the same data from the customerData.